src/components/blockquote/blockquote.twig

```diff
@@ -0,0 +1,37 @@
+{#
+ # Available Props:
+ # - pull_quote__style: bar-left, bar-right, or quote-left
+ #
+ # Available Variables:
+ # - block_quote__quote (required)
+ # - block_quote__attribution
+ # - block_quote__aria_label
+ #}
+
+{% set block_quote__base_class = 'block-quote' %}
+{% set block_quote__style = block_quote__style|default('bar-left') %}
+{% set block_quote__aria_label = block_quote__aria_label|default('Blockquote Element by Lorem Ipsum') %}
+{# If block_quote__attributes is not defined, set it to an empty object by default #}
+{% set block_quote__attributes = block_quote__attributes|default({}) %}
+
+{% set block_quote__attributes = block_quote__attributes|merge({
+  'data-component-width': 'content',
+  'data-block-quote-style': block_quote__style,
+  'aria-label': block_quote__aria_label,
+  class: bem(block_quote__base_class, block_quote__modifiers, block_quote__blockname),
+}) %}
+
+<aside {{ add_attributes(block_quote__attributes) }}>
```

Comment:
@mariannuar I think this should be a `<blockquote>` instead of an `<figure>`, since the text is the main content of this component. If the main content was an image, video, or diagram, then I could see using `figure`.